Lib/test/signalinterproctester.py

```diff
@@ -75,9 +75,12 @@ def test_interprocess_signal(self):
             child.wait()
 
         signal.alarm(1)
-        self.wait_signal(None, 'SIGALRM', KeyboardInterrupt)
-        self.assertEqual(self.got_signals, {'SIGHUP': 1, 'SIGUSR1': 1,
-                                            'SIGALRM': 0})
+        try:
```

Comment:
Move `signal.alarm(1)` in a 'try' block?

Reply:
Done.